dotbot/dispatcher.py

```diff
@@ -18,7 +18,10 @@ def __init__(
     ):
         self._log = Messenger()
         self._setup_context(base_directory, options)
-        plugins = plugins or []
+        if plugins == None:
```

Comment:
Please always use identity, not equality, comparisons with `None`:

```suggestion
        if plugins is None:
```